travis/.rubocop_rspec_base.yml

```diff
@@ -114,10 +114,143 @@ SpaceAroundEqualsInParameterDefault:
 StringLiterals:
   Enabled: false
 
+# This is problematic for Cucumber
+Lint/AmbiguousRegexpLiteral:
+  Exclude:
+    - features/**/*
+
+# In this context, eval is not a "security risk"
+Lint/Eval:
+  Enabled: false
```

Comment:
We have no eval in our lib code but we do in specs. Newer syntactic constructs like kw args can't parse on 1.8.7 and 1.9 so we use eval in some specs that are conditionally run so those things are only parsed on rubies that can handle them. 

However, we clearly want this cop for our lib code. Is there a way to enable it only for lib?